syntaxes/dune.json

```diff
@@ -1235,9 +1235,214 @@
+                              "include": "#general"
+                            }
+                          ]
+                        }
+                      ]
+                    },
+                    {
+                      "include": "#general"
+                    }
+                  ]
+                },
+                {
+                  "comment": "rule/fallback",
+                  "begin": "\\([[:space:]]*(fallback)\\b",
+                  "end": "\\)",
+                  "beginCaptures": {
+                    "1": {
+                      "name": "keyword.language.dune"
+                    }
+                  },
+                  "patterns": [
+                    {
+                      "include": "#general"
+                    }
+                  ]
+                },
+                {
+                  "comment": "rule/locks",
+                  "begin": "\\([[:space:]]*(locks)\\b",
+                  "end": "\\)",
+                  "beginCaptures": {
+                    "1": {
+                      "name": "keyword.language.dune"
+                    }
+                  },
+                  "patterns": [
+                    {
+                      "include": "#general"
+                    }
+                  ]
+                },
+                {
+                  "comment": "rule/alias",
+                  "begin": "\\([[:space:]]*(alias)\\b",
+                  "end": "\\)",
+                  "beginCaptures": {
+                    "1": {
+                      "name": "keyword.language.dune"
+                    }
+                  },
+                  "patterns": [
+                    {
+                      "include": "#general"
+                    }
+                  ]
+                },
+                {
+                  "comment": "rule/package",
+                  "begin": "\\([[:space:]]*(package)\\b",
+                  "end": "\\)",
+                  "beginCaptures": {
+                    "1": {
+                      "name": "keyword.language.dune"
+                    }
+                  },
+                  "contentName": "variable.declaration.other.dune",
+                  "patterns": [
+                    {
+                      "include": "#general"
+                    }
+                  ]
+                },
+                {
+                  "comment": "rule/enabled_if",
+                  "begin": "\\([[:space:]]*(enabled_if)\\b",
+                  "end": "\\)",
+                  "beginCaptures": {
+                    "1": {
+                      "name": "keyword.language.dune"
+                    }
+                  },
+                  "patterns": [
+                    {
+                      "include": "#general"
+                    }
+                  ]
+                },
                 {
                   "include": "#action"
                 },
+                {
+                  "include": "#variables"
+                },
```

Comment:
What's the point of including `#variables` here?

The rules in "variables" are just for highlighting the name in between `%{` and `}`.
The actual syntax for the full variable `%{...}` is already included in `#general`.